python/rascaline/power_spectrum.py

```diff
@@ -0,0 +1,201 @@
+from math import sqrt
+
+import numpy as np
+from equistore import Labels, TensorBlock, TensorMap
+
+
+try:
+    from equistore import slice_block
+except ImportError as err:
+    raise ImportError(
+        "equistore-operations are required to use the power spectrum."
+    ) from err
+
+
+def compute_power_spectrum(
```

Comment:
Do we want this to be a single function, or do we want to follow the same `Calculator` interface as the rest of the code?